<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
		<!-- node parameters -->
		<param name="serial_name" type="string" value="/dev/ttyTHS1"/>
		<param name="baud_rate" type="int" value="230400"/>
		<param name="app_id" type="int" value="1039757"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="Not required in this SDK version"/>
		<param name="enc_key" type="string" value="eaffc890edcb69a5fa2e261c945e7fbf68517fdf4a241c1f13274519ac3c2624"/>
		<param name="groundstation_enable" type="int" value="1"/>
	</node>
</launch>

